docs/reference/analysis/analyzers.asciidoc

```diff
@@ -1,71 +1,148 @@
+
+[source,js]
+--------------------------------------------------
+include::../snippets/analysis/configure-analyzer.json[]
+--------------------------------------------------
+// SENSE: analysis/configure-analyzer.json
+
+Similarly, you can add an analyzer to an existing index by updating the index
+settings. To do this, you need to close the index, add the analyzer, and then reopen the index.
+The following example adds a custom analyzer called `content` that
+uses the `whitespace` tokenizer.
+
+[source,js]
+--------------------------------------------------
+include::../snippets/analysis/add-analyzer.json[]
+--------------------------------------------------
+// SENSE: analysis/add-analyzer.json
+
+NOTE: While you can add analyzers to the index settings, you cannot use the 
+<<indices-update-settings, `update settings`>>  API to modify existing analyzers 
+if you have already indexed documents. If you were to change an existing analyzer, 
+data that had already been indexed wouldn't match the new configuration and your 
+searches would no longer work as expected. To update an analyzer or change which 
+analyzer is used for a particular field, you need to http://www.elasticsearch.org/guide/en/elasticsearch/guide/current/reindex.html[reindex] your data.
 
 [float]
 [[default-analyzers]]
-=== Default Analyzers
+=== Configuring Default Analyzers
+You can configure which analyzers to use by default by configuring either the 
+`default` analyzer or the `default_index` and `default_search` analyzers:
 
-An analyzer is registered under a logical name. It can then be
-referenced from mapping definitions or certain APIs. When none are
-defined, defaults are used. There is an option to define which analyzers
-will be used by default when none can be derived.
+* The `default` analyzer is used for both indexing and searching. 
+* The `default_index` analyzer is used only when indexing.
+* The `default_search` analyzer is be used only when searching.
 
-The `default` logical name allows one to configure an analyzer that will
-be used both for indexing and for searching APIs. The `default_index`
-logical name can be used to configure a default analyzer that will be
-used just when indexing, and the `default_search` can be used to
-configure a default analyzer that will be used just when searching.
+In most cases, you want to use the same analyzer for both indexing and searching,
+so setting the `default` analyzer is recommended. For example:
+
+[source,js]
+--------------------------------------------------
+include::../snippets/analysis/configure-default-analyzer.json[]
+--------------------------------------------------
+// SENSE: analysis/configure-default-analyzer.json
+
+If no default is configured and you don't specify an analyzer when configuring mappings
+or searching, Elasticsearch uses the <<analysis-standard-analyzer, `standard`>> analyzer.
 
 [float]
 [[aliasing-analyzers]]
```

Comment:
I wondered the same thing. (In fact, at one point, there was a comment to that effect in the doc.) I couldn't come up with a reason why it would be useful...